fastai/callbacks/hooks.py

```diff
@@ -99,19 +99,29 @@ def num_features_model(m:nn.Module)->int:
 def total_params(m:nn.Module) -> int:
     params = 0
     if hasattr(m, "weight") and hasattr(m.weight, "size"): params += m.weight.numel()
-    if hasattr(m, "bias") and hasattr(m.bias, "size"):     params += m.bias.numel()
+    if hasattr(m, "bias") and hasattr(m.bias, "size"): params += m.bias.numel()
     return params
 
 def hook_params(modules:Collection[nn.Module]) -> Hooks:
     return Hooks(modules, lambda m, i, o: total_params(m))
 
 def params_size(m: nn.Module, size: tuple = (64, 64)) -> Tuple[Sizes, Tensor, Hooks]:
     "Pass a dummy input through the model to get the various sizes. Returns (res,x,hooks) if `full`"
+    if isinstance(m, Learner):
+        dl = m.data.train_dl
```

Comment:
You have method one_batch in DataBunch that will do this for you now.

Reply:
Thanks!